src/main/java/data/domain/RacingGame.java

```diff
@@ -0,0 +1,41 @@
+package data.domain;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class RacingGame {
+
+    private static final int END_GAME_LAPS = 0;
+    private static final int ENGINE_MAX_POWER = 10;
+    private static final Random RANDOM = new Random();
+
+    private List<Car> cars;
+    private int laps;
+
+    public RacingGame(int participants, int laps) {
+        this.cars = ready(participants);
+        this.laps = laps;
+    }
+
+    private List<Car> ready(int participants) {
+        return Stream.generate(Car::new)
+            .limit(participants)
+            .collect(Collectors.toList());
+    }
+
+    public int [] move() {
+        if(laps == END_GAME_LAPS) {
```

Comment:
```suggestion
        if (laps <= END_GAME_LAPS) {
```

어떠한 이유로 laps가 -1이 된다면 해당 예외처리는 동작하지 않을 수 있습니다!
범위의 검증은 위와 같이 처리하는 습관을 들여보시면 어떨까요?